docs/_templates/base.html

```diff
@@ -115,6 +115,47 @@
     {%- endif %}
   {%- endblock %}
   {%- block extrahead %} {% endblock %}
+
```

Comment:
It would be better to add the `cookie-consent` in its own file in `_docs/_templates/_parts`